src/frontend-samples/clash-review-sample/ClashDetectionApis.ts

```diff
@@ -111,22 +106,21 @@ export default class ClashDetectionApis {
   //   const response = ClashDetectionApis.getValidationUrlResponse(body._links.run.href);
   // https://dev-developer.bentley.com/api-groups/project-delivery/apis/validation/operations/run-validation-test
```

Comment:
Dev URL

Reply:
Not sure what to do about this. What did you have in mind?

Reply:
```suggestion
  // https://developer.bentley.com/api-groups/project-delivery/apis/validation/operations/run-validation-test
```